.dockerignore

```diff
@@ -0,0 +1,11 @@
+node_modules
```

Comment:
According to the task for each excluded folder should be added description. But I think it is minor issue.